frontend/web/src/features/guestdashboard/components/GuestFamilyRow.js

```diff
@@ -0,0 +1,191 @@
+// GuestFamilyRow.jsx
+import React from "react";
+import editIcon from "../../../images/icons/edit-05.png";
+import checkIcon from "../../../images/icons//checkPng.png";
+import { limitBetween, formatFamilyLabel } from "../utils/guestDashboardUtils";
+
+const GuestFamilyRow = ({
+  userFamily,
+  familyCounts,
+  setFamilyCounts,
+  isEdit,
+  onStartEdit,
+  onCancelEdit,
+  onSave,
+}) => {
+  const increase = (key) =>
+    setFamilyCounts((prev) => ({
+      ...prev,
+      [key]: limitBetween(prev[key] + 1),
+    }));
+
+  const decrease = (key) =>
+    setFamilyCounts((prev) => ({
+      ...prev,
+      [key]: limitBetween(prev[key] - 1),
+    }));
+
+  return (
+    <div className="pi-row">
+      <div className="pi-left">
+        <span className="pi-label">Family:</span>
+
+        {!isEdit && (
+          <span className="pi-value" title={userFamily || "-"}>
+            {userFamily || "-"}
+          </span>
+        )}
+
+        {isEdit && (
+          <div
+            className="booking-details__pi"
+            style={{ display: "grid", gap: 8 }}
+          >
```

Comment:
styling must be in the scss file
```suggestion
          <div className="booking-details__pi">
```